utils/mdxUtils.ts

```diff
@@ -44,13 +44,13 @@ export const getAllDocs = async () => {
 
       // Get URL pathname
       const pathname = url.split('/').pop()
-      const folder = url.split('/')[2]
 
       return {
         url,
         title: data.title || pathname.replace(/\-/g, ' '),
+        description: data.description || '',
         nav: data.nav ?? Infinity,
-        content,
+        content: content,
```

Comment:
We'll need to clean this up to maintain code style.

Reply:
Fixed!